test/include/util/storage_test_util.h

```diff
@@ -7,8 +7,10 @@
 #include <vector>
```

Comment:
Signature changes:

`static void CheckInBounds(const A *const val, const B *const lower, const C *const upper)`

`static void CheckNotInBounds(const A *const val, const B *const lower, const C *const upper)`

`static void CheckAlignment(const A *const val, const uint32_t word_size)`

`static A *IncrementByBytes(const A *const ptr, const uint64_t bytes)` this might not work without const_cast. If it's a pain, skip it.

`static storage::BlockLayout RandomLayout(const uint16_t max_cols, Random *const generator)`

`static void FillWithRandomBytes(const uint32_t num_bytes, byte *const out, Random *const generator)`

`static void PopulateRandomRow(storage::ProjectedRow *const row, const storage::BlockLayout &layout, const double null_bias, Random *const generator)`

`static std::vector<uint16_t> ProjectionListRandomColumns(const storage::BlockLayout &layout, Random *const generator)`

`static bool ProjectionListEqual(const storage::BlockLayout &layout, const storage::ProjectedRow *const one, const storage::ProjectedRow *const other)`

`static void InsertTuple(const storage::ProjectedRow &tuple, const storage::TupleAccessStrategy *const tested, const storage::BlockLayout &layout, const storage::TupleSlot slot)`

`static void CheckTupleEqual(const storage::ProjectedRow &expected, const storage::TupleAccessStrategy *const tested, const storage::BlockLayout &layout, const storage::TupleSlot slot)`